docs/reference/how-to/indexing-speed.asciidoc

```diff
@@ -114,13 +114,6 @@ The default is `10%` which is often plenty: for example, if you give the JVM
 10GB of memory, it will give 1GB to the index buffer, which is enough to host
 two shards that are heavily indexing.
 
-[float]
-=== Disable `_field_names`
-
-The <<mapping-field-names-field,`_field_names` field>> introduces some
-index-time overhead, so you might want to disable it if you never need to
-run `exists` queries.
```

Comment:
Just to confirm, this is removed because indexing this field doesn't have the index overhead it once did?

Reply:
Right. @colings86 made things better in #26930 and we now use doc values or norms to run queries most-of-time, and only index into the `_field_names` field when doc values are not enabled (either because the user disabled them or because the field doesn't support doc values).